Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -125,7 +127,15 @@
         'Custom' : 'SLHCUpgradeSimulations/Configuration/combinedCustoms.cust_2023tilted',
         'Era' : 'Phase2C2_timing',
         'ScenToRun' : ['GenSimHLBeamSpotFull','DigiFull','RecoFullGlobal', 'HARVESTFullGlobal'],
+    },
+    '2023D6' : {
+        'Geom' : 'Extended2023D6',
+        'GT' : 'auto:run2_mc',
```

Comment:
everything else upgrade Ph2 is using auto:phase2_realistic.
Should this be changed?